Senla/RPS.swift

```diff
@@ -0,0 +1,93 @@
+import Foundation
+class RPS{
+    enum variants{
```

Comment:
Типы всегда с большой буквы начинают наименование
```suggestion
    enum Variants{
```